resources/js/components/DonateButtons.vue

```diff
@@ -46,8 +46,12 @@ export default {
 	async created ()
 	{
 		this.loading = true;
-		await this.$store.dispatch('GET_DONATION_AMOUNTS');
-		this.loading = false;
+		await this.$store.dispatch('GET_DONATION_AMOUNTS')
+            .then(() => {
+                this.loading = false;
+
+                this.$emit('donations-loaded')
+            });
```

Comment:
No need to call `then` here, it won't run until after await anyway

Reply:
True, missed that. Added a commit for it.